### PR TITLE
#621 Molecule - FMDialog

### PR DESCRIPTION
**Pull request by gilangdarmawan**
### Detailed description of changes

- Add `FMDialog` component
- Add `FODialogWithAlert` component
- Fix FOSidebar menu collapse
- Fix `MuiTab` style

### Recommendations on how to test the changes

- Go to storybook and choose `FMDialog` or `FODialogWithAlert`
- Click the button open modal to show the dialog

### Checklist for developer

- [x] The code builds clean without any errors or warnings
- [x] The code does not contain commented out code
- [x] The code does not log anything to console
- [x] I have added unit test(s) to cover new code and successfully executed ran it
- [x] I have thoroughly tested the new code and any adjacent features it may affect

### Link to related issue(s)

[#621 Molecule - FMDialog](https://app.zenhub.com/workspaces/flick-614155581501e1000f356a5d/issues/hzn-one/tms/621)

#### Figma

Checked
Failed Generating Invoice

#### Flex

localhost_6006_iframe html_id=molecules-dialog--with-title-and-actions args= viewMode=story(Laptop)
localhost_6006_iframe html_id=organisms-dialog-with-alert--default args= viewMode=story(Laptop)

### CODE REVIEW

File: src/Components/Organisms/FODialogWithAlert/FODialogWithAlert.interface.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { ReactNode } from "react";
+import { IFMDialogProps } from "@Molecules/FMDialog/FMDialog.interface";
```
Comment (bryantobing12) [inline src/Components/Organisms/FODialogWithAlert/FODialogWithAlert.interface.ts:2]: please update to relative path

Comment (bryantobing12) [inline src/Components/Organisms/FODialogWithAlert/FODialogWithAlert.interface.ts:2]: please update to relative path

File: src/Components/Organisms/FODialogWithAlert/index.tsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
+import React, { memo } from "react";
+import { Box } from "@mui/system";
```
Comment (bryantobing12) [inline src/Components/Organisms/FODialogWithAlert/index.tsx:2]:
```suggestion
import Box from "@mui/material/Box";
```

File: src/Components/Organisms/FODialogWithAlert/index.tsx [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+          alignItems="center"
+          justifyContent="center"
+          fontSize={iconProps(icon).fontSize}
+          height={64}
+          width={64}
+          mx="auto"
+        >
+          <FAIcon
+            testID="icon-dialog-with-alert"
+            color={iconProps(icon).color as any}
```
Comment (bryantobing12) [inline src/Components/Organisms/FODialogWithAlert/index.tsx:68]: add color in 
```ts
sx={{
 color: iconProps(icon)
}}
```
so we don't have to use `any`

**Review by bryantobing12 — CHANGES_REQUESTED** [review]

**Review by muhammad-farhan64 — APPROVED** [review]

**Review by bryantobing12 — APPROVED** [review]

**Pull request merged**